magento/pwa-studio
# Fix/add jsx loading

## Description

This update adds the ability for people to use `.jsx` files with PWA Studio.
### Verification Steps
1. Create a new component, with `.jsx` file type
2. Import new component into an existing component, _without_ specifying the extension in the import statement, e.g. `import MyComponent from './MyComponent';`, where `MyComponent.jsx` exists alongside the file that the import statement is being added to
3. Verify that webpack doesn't throw an error indicating it can't resolve the path

## Related Issue
Closes #2127.

### Tests

Tests have been updated and run, with no errors
![image](https://user-images.githubusercontent.com/18253422/73409077-26760400-434e-11ea-946e-f17b66ee6426.png)
